.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -1,5 +1,10 @@
 ---
 name: Documentation Issue
+about: Issues, deficiencies or improvements discovered on Solidity's documentation.
```

Comment:
```suggestion
about: Corrections, improvements or requests for new content on Solidity's documentation.
```

Reply:
Whichever one you choose, it should be `improvements discovered in Solidity's documentation.`